src/commons/actor.rs

```diff
@@ -207,15 +208,15 @@ impl Actor {
         A: ToPolar + Display + Clone,
         R: ToPolar + Display + Clone,
     {
-        if let Some(error_msg) = &self.auth_error {
+        if let Some(api_error) = &self.auth_error {
             trace!(
-                "Unable to check access: actor={}, action={}, resource={}: {}",
+                "Unable to check access(#1): actor={}, action={}, resource={}: {}",
```

Comment:
Rather than #1 and #2 we should change this message as it is not about being unable to check access, it's that the process of authenticating the user failed and so is not permitted access. So this should be "Access denied: ..."

Reply:
How about `Authentication denied` since actual access also needs a policy?